pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -755,18 +754,16 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 	for _, refs := range containerMap {
 		if len(refs) == 1 {
 			result = append(result, refs[0])
+			if refs[0].State != runtimeapi.ContainerState_CONTAINER_EXITED {
```

Comment:
Should this be the same condition as below, I.e. container that is running?

Reply:
Yes, probably. My intention was to explicitly only exclude exited containers, but looks like the other container states (unknown and created) should also get excluded